data_summaries/COIG-CQIA.json

```diff
@@ -0,0 +1,50 @@
+{
+    "coig-cqia": {
+        "Unique Dataset Identifier": "coig-cqia",
+        "Dataset Name": "COIG Quality is All You Need",
+        "Paper Title": "COIG-CQIA: Quality is All you need for Chinese Instruction Fine-tuning",
+        "Dataset URL": "https://github.com/paralym/COIG-CQIA",
+        "GitHub URL": "https://github.com/paralym/COIG-CQIA",
+        "Hugging Face URL": "https://huggingface.co/datasets/m-a-p/COIG-CQIA",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "COIG-CQIA",
+        "Collection URL": "https://huggingface.co/datasets/m-a-p/COIG-CQIA",
+        "Languages": [
+            "Chinese"
+        ],
+        "Task Categories": [
+            "Question Answering"
+        ],
```

Comment:
This doesn't need to be perfect, but do you have a sense of what other tasks are covered here? Likely a few others are prominent